kernel/src/syscall/epoll.rs

```diff
@@ -126,9 +140,26 @@ fn do_epoll_wait(
         .as_ref()
         .is_err_and(|err| err.error() == Errno::ETIME)
```

Comment:
Combine this logic into the subsequent match. Unpacking `result` twice is not a good idea.